src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx

```diff
@@ -7,6 +7,49 @@ topics:
 
 Use this document to learn how to review and peer edit docs for you fellow writers in GitHub. Check the [Tech Writer workflow doc](/docs/style-guide/writer-workflow/tech-writer-workflow/) for info on how to set up your local environment. To troubleshoot GitHub issues, see [our guide](/docs/style-guide/writer-workflow/github-troubleshooting/).
 
+## What needs a peer review [#peer-reqs]
+
+Any substantial pull request should be peer reviewed. The line between substantial and trivial is a bit blurry, so refer to the following table when in doubt:
+
+<table>
+<thead>
+<tr>
+<th style={{ width: "200px" }}>
+Doesn't need a peer review
+</th>
+<th>
+Needs a peer review
+</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+Edits less than 5 lines.
+</td>
+<td>
+Edits more than 5 lines.
+</td>
+</tr>
+<tr>
+<td>
+Formatting and small markdown fixes such as fixing links, making a UI component bold, putting a command in a code block, etc.
+</td>
+<td>
+Collapser and table edits.
+</td>
+</tr>
+<tr>
+<td>
+Redirects not related to recent taxonomy changes.
+</td>
+<td>
+Any changes that affect the taxonomy and redirects that result from those changes.
+</td>
+</tr>
+</tbody>
+</table>
+
```

Comment:
One other thing not addressed in this draft was brought up in team meeting: Amount of time for the build to build, plus the amount of time to get a review, plus the amount of time to review. This all adds up to significant time delays, which is one reason why I think our Jira board has been affected by the ability to complete the work. If a PR affects a time-sensitive release and it affects multiple docs, and the Gatsby build works, I am in favor of proceeding without a peer review. But I don't think we reached consensus about the time impact.

Reply:
I think we can use our best judgement in situations like this! One thing @zuluecho9 does is ping me specifically when I'm assigned one of his PRs and he needs to get them out in the next day or so.

Reply:
Basically, I think we can be really active about telling our reviewers when something is a priority or not. If it is a priority and our reviewer is busy, we can just ask the team if anyone is available to review. @austin-schaefer does this all the time.

Reply:
I think it would be helpful to clarify what we mean when we say "PR review" and what we mean when we say "peer edit". I think these things have gotten a bit muddied.

To my mind: 
A PR review is for non-Jira PRs that need a second pair of eyes. I feel like it's ok for this to be a pretty quick pass.
A peer edit is related to a specific Jira and should probably be a bit more in-depth.

Reply:
@paperclypse I really like this distinction.